iroha_data_model/src/lib.rs

```diff
@@ -731,35 +778,37 @@ pub mod asset {
     }
 
     impl Asset {
+        /// Constructor
+        pub fn new<V: Into<AssetValue>>(id: Id, value: V) -> Self {
```

Comment:
And here...

Reply:
Here it is not needed, this structure construction happens only inside Iroha2 automatically.